learnyounode/09-time-server.js

```diff
@@ -0,0 +1,43 @@
+var net = require('net');
+
+var server = net.createServer(function(socket){
+  var date = new Date();
+  data = date.getFullYear()+'-'+('0'+(date.getMonth()+1)).slice(-2)+'-'+('0'+ date.getDate()).slice(-2)+' '+date.getHours()+':'+date.getMinutes() + '\n';
+  socket.end(data); // close the socket
+});
+server.listen(process.argv[2]);
```

Comment:
That approach to zero-fill is pretty unusual, but will work. However, you didn't apply it consistently - hours and minutes need the same treatment.